Demo/Sources/Recycling/GridViews/AdRecommendations/AdRecommendationsGridViewDemoViewHelpers.swift

```diff
@@ -110,7 +112,9 @@ struct AdFactory {
                 scaleImageToFillView: scaleImageToFillView,
                 adType: .normal,
                 sponsoredAdData: index % 4 == 0 ? sponsoredAdData : nil,
-                favoriteButtonAccessibilityLabel: "Sett annonsen som favoritt")
+                badgeViewModel: index == 1 ? badgeViewModel : nil,
```

Comment:
Is it possible to not reference the numerical index? Seems fragile. I would also recommend moving the check to where the model is created to avoid allocating a model when not necessary.

Reply:
I moved the view model out so we create it only once. ✅ 
When it comes to the index, this is demo code and we already make many assumptions about indexes in this code. So it is possible, but maybe not necessary. Then we'd have to change how the entire demo is built. 😄